frontend-vue/src/components/core/navbar/ContextureNavbar.vue

```diff
@@ -15,6 +15,9 @@
           </RouterLink>
         </li>
       </ul>
+      <div class="!ml-auto !mr-5 flex">
```

Comment:
Not a big fan of inverting here.

My suggestion for the impl:

```
  <nav class="bg-blue-500 py-3.5 px-3 text-white sm:px-0" role="navigation">
    <div class="flex items-center justify-between">
      <div class="flex items-center flex-wrap space-x-5 pl-0 sm:pl-10">
        <div>
          <img alt="Contexture Logo" class="h-7 w-7" src="../../../assets/logo/light/light.png" />
        </div>
        <ul class="flex space-x-4">
          <li v-for="item in items" :key="item.to">
            <RouterLink
              :class="{ 'router-link-active': isActiveRoute(item) }"
              :to="item.to"
              class="text-blue-200 hover:text-gray-100"
            >
              {{ item.title }}
            </RouterLink>
          </li>
        </ul>
      </div>
      <div class="mr-5 flex">
        <SignIn />
      </div>
    </div>
  </nav>
```

Reply:
`space-x-5` of the parent element seem to take precedence over `mr-5` hence the use of `!` important atribute

Reply:
I slightly refactored the layout code to create 3 column layout where middle container that displays list of links uses flex-1 to take all available space and pushes the 3rd column to the right as a replacement for using `!ml-auto`

Visually navbar looks the same